### ISSUE TITLE
Slow mapping

### ISSUE THREAD

**Issue by francicco** [alexdobin/STAR#755]
Hi,

I recently change the cluster I'm working with, I installed the git version of STAR and aligned my staff. What I noticed is that the mapping rate drops from something like 140M/h to 33M/h. Is is due to different I/O rate of I did something wrong compiling the software?

Thanks a lot
F

**Comment by Emiller88** [comment]
@francicco I just started using STAR, on a cluster and noticed it's not aligning quickly either. I'm using this wrapper https://snakemake-wrappers.readthedocs.io/en/stable/wrappers/star/align.html . 

Just commenting to let you know I'm happy to help be another test case.

**Comment by alexdobin** [comment]
Hi Francesco, Edmund,

sorry for the belayed reply...
Could you please send me Log.out, Log.progress.out and Log.final.out files from the slow runs?

Cheers
Alex

**Comment by francicco** [comment]
Hi Alex,

Thanks for your reply. Attached the log files. This was particularly slow.
Cheers
F
[Log.out.gz](https://github.com/alexdobin/STAR/files/3775689/Log.out.gz)
[Ptel_4samples_bcfrac1_230M.1.90.curated.tigmint.2LRScaf.1arcs.1Gclos.STARarcs.Log.progress.out.gz](https://github.com/alexdobin/STAR/files/3775816/Ptel_4samples_bcfrac1_230M.1.90.curated.tigmint.2LRScaf.1arcs.1Gclos.STARarcs.Log.progress.out.gz)
[Ptel_4samples_bcfrac1_230M.1.90.curated.tigmint.2LRScaf.1arcs.1Gclos.STARarcs.Log.final.out.gz](https://github.com/alexdobin/STAR/files/3775817/Ptel_4samples_bcfrac1_230M.1.90.curated.tigmint.2LRScaf.1arcs.1Gclos.STARarcs.Log.final.out.gz)

**Comment by Emiller88** [comment]
I believe mine was due to the nodes only having 31G of RAM available so it was writing to swap which was slowing it down.

**Comment by alexdobin** [comment]
Hi Francesco,

this does not seem to be very slow, ~220M reads per hour, for 277b reads, and with 12.44% of "other" unmapped reads, which I would guess is owing to not perfect genome assembly?

Cheers
Alex

**Comment by francicco** [comment]
Sometimes I can map at 400M r/h, but I guess it's due to the fragmentation of the assembly. Is that right?
F

**Comment by alexdobin** [comment]
Right: if you see a large % of reads in the "Unmapped other" category, it points to RNA sequences that have low identity with the reference.
